Python/Product/EnvironmentsList/DBExtension.xaml

```diff
@@ -125,10 +74,47 @@
             <CollectionViewSource x:Key="SortedPackages" Source="{Binding Packages}" />
         </Grid.Resources>
 
-        <ContentControl Content="{Binding}"
-                        ContentTemplate="{StaticResource ProgressView}"
-                        Focusable="False"/>
-        
+        <Grid DataContext="{Binding EnvironmentView}">
```

Comment:
This change may be unnecessary now that I've improved the SetLiveProperty() call - previously you couldn't use it in a DataTemplate because the control was not correctly parented on creation, but that shouldn't matter now.

In any case, I kind of like the new code better, at least now it's written.